src/CaptivePortal.cpp

```diff
@@ -137,13 +262,13 @@ void handleWebSocketClientMessage(std::uint8_t socketId, WStype_t type, std::uin
   }
 
   String typeStr = doc["type"];
-  if (typeStr.length() == 0) {
+  if (typeStr.isEmpty()) {
     ESP_LOGE(TAG, "Message type is missing");
     return;
   }
```

Comment:
Question: is `typeStr == 0` true when the field `type` wasn't even defined? Seems like this could be a null pointer risk. (I don't know!)

Reply:
String is a value type, not a reference or pointer type, meaning it needs to constructed in a valid way for doc[""] to return it.
It will be a empty string, theres no issue here :3